bioio_ome_zarr/writers/utils.py

```diff
@@ -217,3 +223,78 @@ def chunk_size_from_memory_target(
 
     # Return as an immutable tuple
     return tuple(chunk_list)
+
+
+def add_zarr_level(
+    existing_zarr: Union[str, Path],
+    scale_factors: Tuple[float, float, float, float, float],  # (T, C, Z, Y, X)
```

Comment:
do we need scale factors for T & C?

Reply:
I don't know that we need it but I think the implementation is simpler if we have a full 5d tuple.

Reply:
hmm but from a user error perspective does it make sense for them to have T & C?

Reply:
I'd vote to remove it then - seems easy to add back if anyone wants it but might help users avoid error in the meantime (a goal of this repo)